scripts/install.sh

```diff
@@ -3,12 +3,28 @@
 home_dir="gnu-koryavov"
 scipts_dir="scripts"
 
+ubuntu_default_editor="atril"
```

Comment:
It seems default document viewer depends on the de installed. You can use something like

```shell
if [[ $DESKTOP_SESSION == "plasma" ]]; then
    sed -i '/pdfviewer_script/s/=.*\.sh/=okular.sh/' config.conf
fi
```

(and I'm not sure Atril is default ubuntu document viewer)

Reply:
Maybe not default, but it can surely be installed on Ubuntu. DE check makes sense

Reply:
How widely is DESKTOP_SESSION environmental variable is supported? It seems like some of the distributions may not support it.

 https://unix.stackexchange.com/questions/116539/how-to-detect-the-desktop-environment-in-a-bash-script

Reply:
Yeah, this variable is not very widespread, but it looks like there is no much trouble with it on KDE/Gnome, and that's why it's the simpliest way to establish wether there is Okular/Evince/etc already installed (Kubuntu and Ubuntu use Okular by default by the way)

Reply:
But detecting document viewers by running `documentviewer --version` is better obviously (but it's not as easy due to the anti-unification)

Reply:
I think for now we'll leave detection of os as it is, and switch to okular as default editor.